docs/FAQ.md

```diff
@@ -18,14 +18,14 @@ Other than the visual differences, and codecs, there are some other key differen
 
 ![Don't hold Bluefin wrong](https://github.com/user-attachments/assets/6d03851b-5d42-4235-9e1c-3dde7c455946)
 
-You can always do what you want, but Bluefin is designed with a certain workflow in mind, here are some guidelines from the raptor experts:
-
-- Developer environments have nothing to do with the OS! Keep development tools defined per project in devcontainers.json and managed by git. 
-- Stick to flatpaks and homebrew, don’t turn off pure image mode unless you understand the maintenance risks. 
-- Check the xdg-docs for overrides, `/etc`, `/var`, `/usr/local`, `/opt`, and your entire home directory are writable - if you’re [using your UNIX system correctly](https://www.youtube.com/watch?v=JOeY07qKU9c), the read only part is invisible!
-- Isolate the old school jank in a container, mangling packages on your host is for the birds. 
-- Don’t overthink rebooting, just turn off your PC when you’re not using it, the flock will handle the rest.
-- Bluefin performs best on Linux friendly hardware, like the Framework series of laptops. Check hardware compatibility before making the commitment.
+You can always do what you want, but Bluefin is designed with a certain workflow in mind. Here are some guidelines from the raptor experts:
+
+- Containerize your developer environments rather than commingle them with the OS. You can define them per project in a `devcontainers.json` file managed by Git, or you can create custom Distroboxes as needed.
```

Comment:
```suggestion
- Containerize your developer environments rather than commingle them with the OS. You can define them per project in a `devcontainers.json` file managed by Git.
```